Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -241,8 +241,8 @@ def prepare(self,mReader, directories, mode='init'):
             wmsplit['DigiFullPU']=1
             wmsplit['RecoFullPU']=1
             wmsplit['RECOHID11']=1
-            wmsplit['DigiFullTriggerPU_2023D17PU'] = 1 
-            wmsplit['RecoFullGlobalPU_2023D17PU']=1
+            wmsplit['DigiFullTriggerPU_2026D17PU'] = 1 
```

Comment:
@kpedro88 @prebello @zhenhu didn't we drop D17? Do we really still need this?

Reply:
It could probably be deleted, I was focusing on the renaming in this PR.